DllImportGenerator/designs/StructMarshalling.md

```diff
@@ -41,6 +41,17 @@ public class MarshalUsingAttribute : Attribute
      public MarshalUsingAttribute(Type nativeType) {}
 }
 
+[AttributeUsage(AttributeTargets.Struct)]
+public class BlittableTypeIfGenericParametersBlittableAttribute : Attribute
```

Comment:
Can we try and remove this attribute by creating an enum we pass to `BlittableTypeAttribute`.

Reply:
There are also two locations where we have `ttt` and `lll`. Fix?